mmdeploy/backend/sdk/wrapper.py

```diff
@@ -12,8 +12,11 @@ class SDKWrapper(BaseWrapper):
     def __init__(self, model_file, task_name, device):
         super().__init__([])
         creator = getattr(c_api, task_name)
-        # TODO: get device id somewhere
-        self.handle = creator(model_file, device, 0)
+        device_id = parse_device_id(device)
+        device_type = 'cpu' if device_id == -1 else 'cuda'
```

Comment:
Device specific names should not appear in device agnostic code. There will be other device name some (e.g. "npu" from Ascend devices). Prefer splitting device name & index and pass device name as it is.